cmake/std/atdm/ATDMDevEnvSettings.cmake

```diff
@@ -277,6 +277,9 @@ ATDM_SET_CACHE(Phalanx_KOKKOS_DEVICE_TYPE "${ATDM_NODE_TYPE}" CACHE STRING)
 ATDM_SET_CACHE(Phalanx_SHOW_DEPRECATED_WARNINGS OFF CACHE BOOL)
 ATDM_SET_CACHE(Tpetra_INST_CUDA "${ATDM_USE_CUDA}" CACHE BOOL)
 ATDM_SET_CACHE(Tpetra_INST_SERIAL "${ATDM_INST_SERIAL}" CACHE BOOL)
+ATDM_SET_CACHE(Tpetra_INST_INT_INT OFF CACHE BOOL)
```

Comment:
I think this is how SPARC builds anyway.  @micahahoward or @sebrowne might want to comment.